### PR TITLE
Add ReddioEx mainnet Project and Reddiozkvm upcoming project

### PR DESCRIPTION
**Pull request by neilydhan**
More project information can be found at our website at www.reddio.com

### CODE REVIEW

File: packages/config/src/layer2s/reddioex.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  NUGGETS,
+  OPERATOR,
+  RISK_VIEW,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('reddioex')
```
Comment (mateuszradomski) [inline packages/config/src/layer2s/reddioex.ts:26]: If you're using discovery you need to add your project to the `discovery` directory in the `backend` subpackage. The input file is `config.jsonc` and once you run `yarn discover ethereum reddioex` a `discovered.json` will be generated. Please include those two files in this PR!

**Review by mateuszradomski — CHANGES_REQUESTED** [review]
Hi,

Please refer to the comment I've left. If you're going to have any problems using discovery you can always reach us on our public discord server ([link](https://discord.gg/eaVKXPmtWk)).

Make sure that all CI checks are passing, to fix format/lintting issues run `yarn format:fix` and `yarn lint:fix`.

File: packages/config/src/layer2s/reddioex.ts (hunk 1 above)
Comment (neilydhan) [reply packages/config/src/layer2s/reddioex.ts:26]: @mateuszradomski sure, just made the changes and committed already, thanks for the review and help.

**Comment by lucadonnoh** [comment]
moved to https://github.com/l2beat/l2beat/pull/2482 since i cannot push

**Pull request closed without merging**